tests/test_menu_continue_stream_1195.py

```
@@ -50,7 +48,9 @@ def __init__(self, fresh: bool = False, on_stage=None) -> None:
         def state_payload(self) -> dict:
             return self._state
 
+    _reset_path_leases()
     monkeypatch.setattr(web_app, "_has_main_db", lambda: True)
+    monkeypatch.setattr(web_app, "_get_main_db_path", lambda: "/tmp/continue-main.db")
```

Ruff (0.16.3)

[error] Probable insecure usage of temporary file or directory: "/tmp/continue-main.db"

(S108)